extensions/amp-a4a/amp-ad-refresh.md

```diff
@@ -0,0 +1,65 @@
+<!---
```

Comment:
Let's move this to amp-ad-network-doubleclick-impl md.  We should probably also indicate that this is still in beta, meaning we don't yet verify its correctness.